ee/tables/nix_env/upgradeable/upgradeable_test.go

```diff
@@ -0,0 +1,69 @@
+//go:build linux
+// +build linux
+
+package nix_env_upgradeable
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestQueries(t *testing.T) {
```

Comment:
I find myself wondering what we're actually testing here. The table is doing an exec with an effective UID, and then sending the xml straight to dataflatten.

Dataflatten has some pretty extensive tests, and if we're stubbing the exec anyhow, what's there to test? I'd probably be comfortable yeeting the tests. Not sure how the rest of launcher feels